src/main/resources/gcc-init.sh

```diff
@@ -154,16 +154,51 @@ format_disks() {
   /usr/local/disk_mount.sh
 }
 
+restart() {
+    if docker inspect ambari-server &> /dev/null ; then
```

Comment:
I think we don't want to make any cloud specific thing in the init script. Please try to avoid this changes.